packages/azure-http-specs/specs/client/namespace/client.tsp

```diff
@@ -0,0 +1,31 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Spector;
+using Azure.ClientGenerator.Core;
+using Client.ClientNamespace;
+
+@route("/client/client-namespace")
+namespace ClientNameSpaceClient;
+
+@client({
+  name: "ClientNamespaceFirstClient",
+  service: Client.ClientNamespace,
+})
+@clientNamespace("client.clientnamespace", "java")
```

Comment:
is it possible to remove the scope? i believe every language needs to test ns customization.

Reply:
Every language would need to use slight different SDK namespace. E.g. csharp would use maybe `Client.ClientNamespace`.

So I think whenever we had `@clientNamespace`, it would be one line for one language.

Reply:
@msyyc could you help to add python part?

Reply:
updated for python.